client/blocks/login/social.jsx

```diff
@@ -44,6 +68,11 @@ class SocialLoginForm extends Component {
 						clientId={ config( 'google_oauth_client_id' ) }
 						responseHandler={ this.handleGoogleResponse } />
 				</div>
+
+				{ this.state.bearerToken &&
+					<WpcomLoginForm log={ this.state.username }
+									authorization={ 'Bearer ' + this.state.bearerToken }
+									redirectTo="/start" /> }
```

Comment:
It is off because there are extra tab characters. You can see that in the raw file: https://raw.githubusercontent.com/Automattic/wp-calypso/df0597bc9a3fadf67e9b16ded42350a35de30770/client/blocks/login/social.jsx